### PR TITLE
[new locale] nl-be: Dutch (Belgium) locale

### PR DESCRIPTION
**Pull request by kevinzwhuang**
These changes address issue https://github.com/moment/moment/issues/3410:
- Renames nl locale to **nl-nl** for Dutch (**Netherlands**) locale
- Creates **nl-be** for Dutch (**Belgium**) locale

_**References for nl-be locale:**_
- Locale Planet reference: http://www.localeplanet.com/java/nl-BE/
- IBM Documentation of nl-be: http://www.ibm.com/support/knowledgecenter/SSS28S_3.0.0/com.ibm.help.forms.doc/locale_spec/i_xfdl_r_formats_nl_BE.html

_**References for nl-nl locale:**_
- Locale Planet nl-nl reference: http://www.localeplanet.com/java/nl-NL/
- IBM Documentation of nl-nl: http://www.ibm.com/support/knowledgecenter/SSS28S_3.0.0/com.ibm.help.forms.doc/locale_spec/i_xfdl_r_formats_nl_NL.html

How does this look to you @jorisroling, @middagj?

### CODE REVIEW

File: src/locale/nl-be.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+var monthsShortWithDots = 'jan._feb._mrt._apr._mei_jun._jul._aug._sep._okt._nov._dec.'.split('_'),
+    monthsShortWithoutDots = 'jan_feb_mrt_apr_mei_jun_jul_aug_sep_okt_nov_dec'.split('_');
+
+var monthsParse = [/^jan/i, /^feb/i, /^maart|mrt.?$/i, /^apr/i, /^mei$/i, /^jun[i.]?$/i, /^jul[i.]?$/i, /^aug/i, /^sep/i, /^okt/i, /^nov/i, /^dec/i];
+var monthsRegex = /^(januari|februari|maart|april|mei|april|ju[nl]i|augustus|september|oktober|november|december|jan\.?|feb\.?|mrt\.?|apr\.?|ju[nl]\.?|aug\.?|sep\.?|okt\.?|nov\.?|dec\.?)/i;
+
+export default moment.defineLocale('nl-be', {
+    months : 'januari_februari_maart_april_mei_juni_juli_augustus_september_oktober_november_december'.split('_'),
+    monthsShort : function (m, format) {
+        if (/-MMM-/.test(format)) {
```
Comment (middagj) [inline src/locale/nl-be.js]: I would then also change this to
```javascript
if (/\/MMM\//.test(format)) {
```

**Comment by middagj** [comment]
I am Dutch (as in nl-nl), so I can't really comment on the standard Belgian way of writing dates. I do see it sometimes like that, and the linked sources do confirm. Is it possible to have a common ancestor of the two files?

(Sorry about the line comment, only if the month is numeric slashes are used so no need to change the if statement.)

**Comment by kevinzwh** [comment]
I just added a commit to restore the original src/locale/nl.js & src/test/locale/nl.js to serve as a common ancestor for both nl-nl & nl-be, does that work @middagj?

**Comment by middagj** [comment]
Oh that is also nice, for if you don't care about NL or BE. I was actually more referring to duplication in the code. I don't know if that is easily done in the current framework.

**Comment by kevinzwh** [comment]
Yeah, duplicating it as NL for now is probably the way to go. AR and the
other AR locales ( AR-LY, AR-MA, ...etc ) also follow this pattern ( see
ar.js, ar-ly.js, etc. )

This is nice too since anyone using NL currently will not suffer a breaking
change from losing the locale to NL-NL.

**Comment by kevinzwh** [comment]
Thoughts on this, @ichernev?

**Comment by kevinzwh** [comment]
Revisiting this PR, does this look good to you @ichernev?

**Comment by kevinzwh** [comment]
Revisiting this PR after a while, @mj1856 how does this look to you?

**Comment by mj1856** [comment]
You don't need to do anything with `nl`, nor do you need a `nl-nl` copy.  Just add the one file `nl-be.js` (and its tests).

Consider that `moment.locale('nl-anything')` will currently use `nl`.

**Comment by kevinzwh** [comment]
Thanks @mj1856 , I went ahead and removed `nl-nl`. Now the only addition will be `nl-be`

**Comment by kevinzwh** [comment]
Hey @mj1856 how does this look now?

**Comment by kevinzwhuang** [comment]
Does this PR look good to you @mj1856 ?

**Comment by ichernev** [comment]
@kevinzwhuang code looks great. You'd also need to sing the CLA (a new thing). This will go out in next release. Thanks for contributing!

**Comment by kevinzwhuang** [comment]
Thanks @ichernev, just signed the CLA!

**Comment by ichernev** [comment]
Merged in 0ad30553f6fd057a8dc6faeb0e434633d0f516c5

**Pull request closed without merging**